packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -500,15 +500,11 @@ export interface CallWithChatAdapterManagement {
   /* @conditional-compile-remove(spotlight) */
   /**
    * Start spotlight
-   *
-   * @beta
    */
   startSpotlight(userId: string): Promise<void>;
   /* @conditional-compile-remove(spotlight) */
   /**
    * Stop spotlight
-   *
-   * @beta
```

Comment:
nit: these should be removed in the same PR that the feature is bumped to stable